### DISCUSSION TITLE
[Idea] Add a choice type to product customization fields

### DISCUSSION THREAD

**Discussion by web-cooking-factory** [PrestaShop/PrestaShop#31713] [Ideas]
### The problem

Sometimes a merchant can be willing to set a customization field for a product based on a specific list of values.

### The solution

Add a choice type to customization field and a textarea to set the list.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [x] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
I like the idea :)
